SwrveConversationSDK/src/main/java/com/swrve/sdk/conversations/ui/ConversationButton.java

```diff
@@ -31,6 +35,11 @@ public ConversationButton(Context context, ButtonControl model, int defStyle) {
         initBackgroundColorStates();
         setLines(1);
         setEllipsize(TextUtils.TruncateAt.END);
+        if (conversationVersion >= 4) { // text/font styling added in v4
```

Comment:
Style should have values for those new properties or null / not set. The button should not know about the conversation version that caused those style changes in my opinion.